packages/@aws-cdk/toolkit-lib/lib/api/aws-auth/sdk.ts

```diff
@@ -88,6 +88,14 @@ import type {
   UpdateTerminationProtectionCommandInput,
   UpdateTerminationProtectionCommandOutput,
   StackSummary,
+  DescribeStackDriftDetectionStatusCommandInput,
+  DescribeStackDriftDetectionStatusCommandOutput,
+  DescribeStackResourceDriftsCommandOutput,
+  DetectStackDriftCommandInput,
+  DetectStackDriftCommandOutput,
+  DetectStackResourceDriftCommandInput,
+  DetectStackResourceDriftCommandOutput,
+  DescribeStackResourceDriftsCommandInput,
```

Comment:
These surely will require new permissions? If not can you make a note why? Or are you preparing a corresponding change to the bootstrap stack?

Reply:
They will need new permissions, yes. Specifically, after looking into it, it seems I'll need to add these:
```
cloudformation:DescribeStackDriftDetectionStatus
cloudformation:DescribeStackResourceDrifts
cloudformation:DetectStackDrift
cloudformation:DetectStackResourceDrift
```
I am preparing to make those changes to the PR as well to update the bootstrap stack.

Reply:
Testing this locally, it looks like I don't need to add these permissions to give access for drift detection. Do you know what role `cdk diff` is using to execute these changes? It may be that it either has these permissions already or is using the admin role.

Reply:
I think `cdk diff` might be using the CLI's credentials, or perhaps the lookup role.

Reply:
Confirmed with AppSec, this uses the lookup role, which has the following permissions:
```yml
...
"cloudformation:Describe*",
"cloudformation:Detect*",
...
```
Hence, no bootstrap changes are necessary.